deploy-manage/uninstall/delete-a-cloud-deployment.md

```diff
@@ -90,3 +90,30 @@ To restore a terminated deployment,
     Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
 3. In the **Deployment Management** section, select **Restore** and then acknowledge the confirmation message.
 
+## {{eck}} [elastic-cloud-kubernetes]
+
+To delete a deployment managed by {{eck}}, remove the corresponding `{{es}}`, `{{kib}}`, and any other related custom resources from your Kubernetes cluster. This action deletes all associated pods and their persistent data.
+To delete an Elasticsearch deployment created with {{eck}}:
```

Comment:
```suggestion

To delete an {{es}} deployment created with {{eck}}:
```

Reply:
“Deployment” is a bit of an overloaded term in Kubernetes, also I don't think we have the concept of "deployment" for ECK since each application can be managed individually?

Reply:
I've reviewed the rest and I think just using here `{{es}} cluster` would be better, as @barkbay recommends.

In the initial introductory sentence, we can leave `To delete a deployment managed by {{eck}}` untouched, as this doc is a generic doc used for `orchestrated deployments`, understanding by orchestrated deployments the Elasticsearch clusters with their corresponding Kibana instances managed by ECE, ECH or ECK, which is something covered in the deploy and general overview docs.